experiment/process-e2e-suite-log/example_report.csv

```diff
@@ -0,0 +1,705 @@
+res_100.tmp,[sig-api-machinery] Watchers   should observe add update and delete events on pods,,Y,Y,,,Y,Y,,,,,,,Y,,Y,,,Y,,,,,Y,,,Y
+res_101.tmp,[sig-storage] Subpath [Volume type: nfs]   should fail if non-existent subpath is outside the volume [Slow],,Y,Y,,,Y,Y,,,,,,,Y,,Y,,,Y,,,,,,,,Y
+res_10.tmp,[sig-storage] Subpath [Volume type: nfs]   should unmount if pod is force deleted while kubelet is down [Disruptive][Slow],,Y,Y,,,Y,Y,,,,,,Y,Y,,Y,,,Y,,,,,,,,Y
+res_102.tmp,[sig-storage] Subpath [Volume type: nfs]   should unmount if pod is gracefully deleted while kubelet is down [Disruptive][Slow],,Y,Y,,,Y,Y,,,,,,Y,,,Y,,,Y,,,,,,,,Y
+res_103.tmp,[sig-storage] Subpath Atomic writer volumes   should support subpaths with downward pod,,Y,Y,,,Y,Y,Y,,,,,,,,,,,Y,,,,,,,,Y
+res_104.tmp,[sig-scheduling] Multi-AZ Cluster Volumes [sig-storage]   should only be allowed to provision PDs in zones where nodes exist,,Y,,,,Y,Y,,,,,,,,,,,,,,,,,,,,
+res_105.tmp,[sig-storage] PersistentVolumes-local  [Volume type: tmpfs] Two pods mounting a local volume one after the other   should be able to write from pod1 and read from pod2,,Y,Y,,,Y,Y,,,,,,,,,Y,,,Y,,,,,,,,Y
+res_106.tmp,[sig-storage] Subpath [Volume type: hostPath]   should support readOnly directory specified in the volumeSource,,Y,Y,,,Y,Y,,,,,,,,,Y,,,,,,,,,,,
+res_108.tmp,[sig-network] Services   should have session affinity work for service with type clusterIP,,Y,Y,,,Y,Y,,,,,,,Y,,Y,,,Y,,,,,,,,Y
+res_109.tmp,[sig-apps] Deployment   RecreateDeployment should delete old pods and create new ones,,Y,Y,,,Y,Y,,,,,,,,,,,,Y,,,,,,,,Y
+res_110.tmp,[sig-api-machinery] Generated clientset   should create v1beta1 cronJobs delete cronJobs watch cronJobs,,Y,,,,Y,Y,,,,,,,Y,,,,,,,,,,,,,
+res_11.tmp,[sig-storage] EmptyDir volumes   should support (root0666tmpfs) [Conformance],Y,Y,Y,,,Y,Y,Y,,,,,,,,,,,Y,,,,,,,,Y
+res_111.tmp,[sig-storage] Volume Placement   should create and delete pod with multiple volumes from different datastore,,Y,,,,Y,Y,,,,,,,,,,,,,,,,,,,,
+res_113.tmp,[sig-storage] Subpath [Volume type: hostPath]   should support existing single file,,Y,Y,,,Y,Y,Y,,,,,,,,Y,,,Y,,,,,,,,Y
+res_112.tmp,[sig-storage] [Serial] Volume metrics PVController   should create none metrics for pvc controller before creating any PV or PVC,,Y,,,,Y,Y,,,,Y,,,,,,,,,,,,,,,,
+res_114.tmp,[sig-storage] Subpath [Volume type: hostPathSymlink]   should support creating multiple subpath from same volumes [Slow],,Y,Y,,,Y,Y,Y,,,,,,,,,,,Y,,,,,,,,Y
+res_115.tmp,[sig-storage] Downward API volume   should provide container's cpu limit  [Conformance],Y,Y,Y,,,Y,Y,Y,,,,,,,,Y,,,Y,,,,,,,,Y
+res_116.tmp,[k8s.io] Pods   should get a host IP  [Conformance],Y,Y,Y,,,Y,Y,,,,,,,,,Y,,,Y,,,,,,,,Y
+res_118.tmp,[sig-storage] PersistentVolumes-local  Local volume that cannot be mounted [Slow]   should fail due to non-existent path,,Y,Y,,,Y,Y,,,,,,,,,Y,,,Y,,,,,,,,Y
```

Comment:
I think we get the idea. I don't think the next 600+ lines add much value for the example

Reply:
Initially left the original file in case anybody wanted to play around or see how it looked like, but, yes... totally. 
Placed a smaller file for example, alongside the resulting report.